Modules/Microsoft365DSC/DSCResources/MSFT_AADConditionalAccessPolicy/MSFT_AADConditionalAccessPolicy.psm1

```diff
@@ -941,7 +941,7 @@ function Set-TargetResource
         {
             $conditions.Applications.Add('IncludeApplications', $IncludeApplications)
         }
-        if ($ExcludeApplications)
+        if ($currentParameters.ContainsKey("ExcludeApplications"))
```

Comment:
Shouldn't this method also be applied to line 940 (IncludeApplications parameter), 948 (IncludeUserActions parameter) and 952 (AuthenticationContexts parameter)??

Reply:
For the attribute IncludeApplication this change is not needed because the default value is empty is "none". With "none" present value are deleted. This was not possible with ExcludeApplication, because the missing "none" as default.